src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -383,15 +383,12 @@ cdef class Buffer:
 
 cdef class Blob:
     cdef hb_blob_t* _hb_blob
-    cdef object _data
 
     def __cinit__(self, bytes data = None):
         if data is not None:
-            self._data = data
             self._hb_blob = hb_blob_create(
                 data, len(data), HB_MEMORY_MODE_READONLY, NULL, NULL)
```

Comment:
```suggestion
                data, len(data), HB_MEMORY_MODE_DUPLICATE, NULL, NULL)
```

Since we are no longer keeping `data` around, so we need to ask HarfBuzz to make a copy.